website/source/api/acl-legacy.html.md

```diff
@@ -347,8 +349,8 @@ This endpoint returns the status of the ACL replication process in the
 datacenter. This is intended to be used by operators, or by automation checking
 the health of ACL replication.
 
-Please see the [ACL Guide](/docs/guides/acl.html#replication) replication
-section for more details.
+Please see the agent configuration [documentation](/docs/agent/options.html#acl_enable_token_replication) for replication
```

Comment:
This is the wrong link. You can have replication without token replication (just replicating policies and roles).

One of these is likely more suitable. They are both for configuring the token used for _doing_ replication:

* `/docs/agent/options.html#acl_tokens_replication`
* `/docs/agent/options.html#enable_acl_replication`

Reply:
Updated to the new ACL Replication guide.